Cargo.lock

```diff
@@ -319,6 +319,7 @@ dependencies = [
  "serde",
  "serde_json",
  "sha2",
+ "similar",
```

Comment:
TIL this was already in our dependency tree. Only via `insta` though so I guess it wasn't in release builds.

Reply:
yeah i was just like "ok what the fuck does insta use" and the answer was "a library they made for insta and gloriously spun off as a really nice standalone project"